scripts/wood_density.py

```diff
@@ -37,6 +37,11 @@ def __init__(self, **kwargs):
         2. Provide recognition of the efforts of this group in the assembly of the data by using the citation for the database above.
         3. Recognize that these data were assembled by the group for various analyses and research questions. If any of these uses overlap with your interests, you recognize that group has precedence in addressing these questions."""
 
+        if VERSION < 2.1.dev:
```

Comment:
I am not sure which method of the two is better. But you can test them as see what you want to be included.